### DISCUSSION TITLE
Exportable Interactive Coverage Browser

### DISCUSSION THREAD

**Discussion by coopershawna** [stuart-lab/signac#616] [Q&A] [answered]
Is it possible to generate an exportable version of the interactive coverage browser so that a non-R literate colleague could explore my data for particular chromatin regions as well?

Thanks!

**Comment by timoast** [comment, marked as answer]
If you're thinking of a web app type of interface, we don't have anything like that currently. Something like cellxgene with the ability to show dynamic coverage tracks would be amazing, but probably not something we'd try to add to Signac (R isn't great for that type of thing).

I think the level of R-literacy needed to run the `CoverageBrowser()` function is quite low, you really just need to load the Seurat object and launch the browser by calling the function. To make things even easier for collaborators, you can host the fragment file on a remote server (anything accessible by http or ftp) so that you can move the saved Seurat objects to different machines without losing access to the fragment file information.
